lib/streamlit/elements/exception_proto.py

```diff
@@ -144,8 +144,6 @@ def _is_in_streamlit_package(file):
 
 def _get_stackframe_filename(frame):
     """Return the filename component of a traceback frame."""
-    # Python 3 has a frame.filename variable, but frames in
```

Comment:
I think we should take advantage of what Python 3 allows and change the code to `return frame.filename`, since we don't need to stick to a harder-to-read construct that also worked with Python 2.

Reply:
Thanks!